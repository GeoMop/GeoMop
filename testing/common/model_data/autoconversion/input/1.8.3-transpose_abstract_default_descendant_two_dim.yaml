flow123d_version: 1.8.3
problem: !SequentialCoupling
  description: Test
  mesh:
    mesh_file: mesh.msh
  primary_equation: !Steady_MH
  
    # Test data
    input_fields:
      # Transpose input_fields to array with two items
      rid: [1, 2]
      # and convert anisotropy values into two dimensional arrays
      anisotropy:
        - [1, 2, 3]
        - [4, 5, 6]


    # Placeholder data
    solver: !Petsc
        r_tol: 1
        a_tol: 1
    balance: 
      balance_on: true
    output:
      output_stream: 
        file: ./test1.pvd
        format: !gmsh
      output_fields:
        - pressure_diff
