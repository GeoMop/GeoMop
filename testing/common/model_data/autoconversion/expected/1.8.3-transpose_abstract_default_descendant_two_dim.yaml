flow123d_version: 1.8.3
problem: !SequentialCoupling
  description: Test
  mesh:
    mesh_file: mesh.msh
  primary_equation: !Steady_MH
  
    # Test data
    input_fields:
      # Transposed input_fields to array with two items
      - rid: 1
        # and converted anisotropy values into two dimensional arrays
        anisotropy: !FieldConstant
          value:
            - [1]
            - [2]
            - [3]
              
      - rid: 2
        anisotropy: !FieldConstant
          value:
            - [4]
            - [5]
            - [6]


    # Placeholder data
    solver: !Petsc
        r_tol: 1
        a_tol: 1
    balance: 
      balance_on: true
    output:
      output_stream: 
        file: ./test1.pvd
        format: !gmsh
      output_fields:
        - pressure_diff
